### PR TITLE
Create/Edit Post

### PR DESCRIPTION
**Pull request by mecrowley**
1. Modified `PostForm.js` to include checkboxes to add tags while creating or editing a post

STEPS TO TEST

1.  checkout to branch mec-create-post2 on both the api and client rare directories
2. log into Rare
3. Click on 'Add Post'
4. At the bottom of the form, checkboxes should display with corresponding tag labels
5. Fill in all input, select some random tag checkboxes, and save post
6. New Post should be created with selected tags
7. When editing a post, tag checkboxes should appear checked already for the tags already added to the post
8. The server does not currently support saving edited tags, so do not expect any changes there to be reflected after saving post (all other details should update)

### CODE REVIEW

**Review by enbeec — APPROVED** [review]
Love it!

File: src/components/post/PostForm.js [hunk 1]
```diff
@@ -94,6 +111,26 @@ export const PostForm = () => {
 
             <fieldset>
                 <div className="form-group tags">
+                    <h4>Tags</h4>
+                    {tags.map(t => {
+                        const foundTag = postTags.find(pt => pt.id === t.id)
+                        if (foundTag) {
+                            return (
+                                <>
+                                    <input type="checkbox" id={t.id} required autoFocus value={t.label} onChange={handleTagInputChange}
```
Comment (enbeec) [inline src/components/post/PostForm.js:120]: I think these need keys?

**Review by enbeec — COMMENTED** [review]
Took a second look at tags because I noticed that they weren't coming through after creating a new post.

**Review by alambrow — APPROVED** [review]
Tested and working as specified!

**Pull request merged**